src/extras/CoolSDS011.cpp

```diff
@@ -54,7 +54,11 @@ bool SDS011::read() {
   Wire.endTransmission();
   delay(200);
   Wire.requestFrom(SDS011_ADRESS, 8);
+  uint32_t temp = millis();
```

Comment:
what ?

Reply:
add timeout on sds011 read

Reply:
OK - but do not name it `temp` ! we use `temp` for `temp`erature already ! just `timeout = millis() + 2000` and compare next millis to timeout instead of doing the addition each time